ci/update-jetpack.js

```diff
@@ -60,22 +60,29 @@ function compareVersions(a,b) {
     return aParts[2] - bParts[2];
 }
 
-function incrementPatchVersion(version) {
-    if (version === 'beta') {
+const MAX_BETA = 10;
```

Comment:
Do we need a maximum beta constraint? It looks like they don't really go above 3 typically.

Reply:
You are right. I refactor the code to not have any limit.